livre/lib/data/MemoryDataSource.cpp

```diff
@@ -88,21 +88,27 @@ MemoryUnitPtr MemoryDataSource::getData( const LODNode& node )
                             _volumeInfo.getBytesPerVoxel();
     const Identifier nodeID = node.getNodeId().getId();
     const uint8_t* id = reinterpret_cast< const uint8_t* >( &nodeID );
-    const uint8_t fillValue = ( id[0] ^ id[1] ^ id[2] ^ id[3] ) + 16 +
+    const uint8_t opacity =  ( id[0] ^ id[1] ^ id[2] ^ id[3] ) + 16 +
```

Comment:
:) People ! dont spend time with it. Lets make it "value". void pointer for a voxel ! I am fine with that.